### PR TITLE
User can change password

### PR DESCRIPTION
**Pull request by Baonganbui123**

### CODE REVIEW

File: src/test/java/tests/ChangePasswordTests.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    String newPassword = "12345678";
+    String successMsg = "Your password has been updated!";
+
+    @Test(description = "User can change password")
+    public void tc009_ChangePasswordWithValidInfo() {
+        LogHelpers.printInfo("Click on 'Register' tab");
+        registerPage.clickRegisterTab();
+
+        LogHelpers.printInfo("Enter valid information into all fields");
+        registerPage.register(newEmail, Constants.VALID_PASSWORD, Constants.VALID_PASSWORD, Constants.VALID_PID);
```
Comment (tanmle) [inline src/test/java/tests/ChangePasswordTests.java:27]: register is precondition

File: src/test/java/tests/RegisterTests.java [hunk 2]
```diff
@@ -29,4 +29,5 @@ public void t007_RegisterWithValidInfo() {
         LogHelpers.printInfo("Verify that success message displayed");
         Assert.assertEquals(registerPage.getSuccessMsg(), successMsg, "Success message doesn't display correctly");
     }
+
```
Comment (MinhQuan119) [inline src/test/java/tests/RegisterTests.java:32]: Redundant row

File: src/test/java/tests/LoginTests.java [hunk 3]
```diff
@@ -1,13 +1,15 @@
 package tests;
 
 import common.Constants;
+import helpers.DataHelpers;
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java:4]: Unused import

File: src/test/java/tests/ChangePasswordTests.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
+package tests;
```
Comment (anhlvt) [inline src/test/java/tests/ChangePasswordTests.java:1]: add test to test suite

File: src/test/java/tests/ChangePasswordTests.java (hunk 1 above)
Comment (Baonganbui123) [reply src/test/java/tests/ChangePasswordTests.java:28]: I changed it, please check again

File: src/test/java/tests/RegisterTests.java (hunk 2 above)
Comment (Baonganbui123) [reply src/test/java/tests/RegisterTests.java]: I removed it, please check again

File: src/test/java/tests/LoginTests.java (hunk 3 above)
Comment (Baonganbui123) [reply src/test/java/tests/LoginTests.java]: I removed it, please check again

File: src/test/java/tests/LoginTests.java [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
 import helpers.LogHelpers;
 import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import page_object.ChangePasswordPage;
 import page_object.LoginPage;
 import page_object.MyTicketPage;
+import page_object.RegisterPage;
```
[the remark this answers is not in the document]
Comment (Baonganbui123) [reply src/test/java/tests/LoginTests.java]: I removed it, please check again

File: src/test/java/tests/ChangePasswordTests.java (hunk 4 above)
Comment (Baonganbui123) [reply src/test/java/tests/ChangePasswordTests.java:1]: I added test to test suite, please check again

**Review by tanmle — APPROVED** [review]

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**